### PR TITLE
Qt 5.11.1 bump

### PR DESCRIPTION
**Pull request by a17r**
See also: https://bugreports.qt.io/browse/QTBUG-68224

Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

**Comment by Pesa** [comment]
does it make sense to keep backporting fixes from 5.11.1 given that it will be released next week?

**Comment by a17r** [comment]
Oh, will it indeed? If we can trust upstream there, then we can probably do without.

**Comment by Pesa** [comment]
I'm reasonably sure they want to release `.1` before the summer break. And the last few releases have been relatively on time (±1 week...)

**Comment by Pesa** [comment]
http://lists.qt-project.org/pipermail/releasing/2018-June/004653.html

**Comment by a17r** [comment]
Added disable-xml-catalogs patch from Arfrever for testing.

**Comment by a17r** [comment]
should we adopt the opensuse patch to disable gpu in qtwebengine if nouveau? https://github.com/OpenMandrivaAssociation/qt5-qtwebengine/blob/master/disable-gpu-when-using-nouveau-boo-1005323.diff

**Comment by okias** [comment]
@Pesa time to merge  :+1:  ?

File: dev-qt/qtwebengine/qtwebengine-5.11.1.ebuild [hunk 1]
```diff
@@ -79,6 +79,8 @@ DEPEND="${RDEPEND}
 	pax_kernel? ( sys-apps/elfix )
 "
 
+PATCHES+=( "${FILESDIR}/${P}-libxml2-disable-catalogs.patch" )
```
Comment (Pesa) [inline dev-qt/qtwebengine/qtwebengine-5.11.1.ebuild]: can you reference a bug# either here or in the commit msg?

File: dev-qt/qtwebengine/files/qtwebengine-5.11.1-ffmpeg4.patch [hunk 2]
```diff
@@ -0,0 +1,32 @@
+From a568ded46a678eac8139cb06595819c5ae874177 Mon Sep 17 00:00:00 2001
+From: Xiaohan Wang <xhwang@chromium.org>
+Date: Mon, 29 Jan 2018 21:28:28 +0000
+Subject: [PATCH] media: Increase DecoderBuffer::kPaddingSize to 64
```
Comment (Pesa) [inline dev-qt/qtwebengine/files/qtwebengine-5.11.1-ffmpeg4.patch]: is this backward compatible with ffmpeg-3?

File: dev-qt/qtwebengine/qtwebengine-5.11.1.ebuild (hunk 1 above)
Comment (Pesa) [reply dev-qt/qtwebengine/qtwebengine-5.11.1.ebuild]: not sure why github shows this diff as "outdated"...

**Comment by a17r** [comment]
> only the first commit in the series is signed?

must have been caused by rebasing...

**Comment by okias** [comment]
for qtwidgets I get: ERROR: Feature 'gtk3' was enabled, but the pre-condition 'features.glib && libs.gtk3' failed.

I'll try update glib before and lets see

**Comment by okias** [comment]
works fine.

Tested-by: David Heidelberg <david@ixit.cz>

File: dev-qt/qtwebengine/qtwebengine-5.11.1.ebuild (hunk 1 above)
Comment (Pesa) [reply dev-qt/qtwebengine/qtwebengine-5.11.1.ebuild]: ping on this, the rest LGTM

Comment (a17r) [reply dev-qt/qtwebengine/qtwebengine-5.11.1.ebuild]: thanks, done!

**Pull request merged**